### DISCUSSION TITLE
New patch WIP to add Send to Streams

### DISCUSSION THREAD

**Discussion by nappa85** [SeaQL/sea-orm#612] [Show and tell]
Hello guys,
I'm actually working on a patch to add Send to Streams.
I've worked out the limitations and found out in ouroboros the root cause.
I've just open a MR to solve the problem https://github.com/joshua-maros/ouroboros/pull/56/
If it's approved, I'll open a MR here with the patch

**Comment by billy1624** [comment]
Hey @nappa85, sorry for the delay. Is this related?
- #622

**Reply by nappa85** [reply]
Yes, as I told on the MR, I didn't know you were already working in it.
Ouroboros is patched, but I've seen that you've stopped using the async builder, so the patch isn't needed anymore

**Reply by billy1624** [reply]
Thanks!! @nappa85
